core/trino-main/src/main/java/io/trino/operator/OperatorInfo.java

```diff
@@ -32,7 +33,8 @@
         @JsonSubTypes.Type(value = PartitionedOutputInfo.class, name = "partitionedOutput"),
         @JsonSubTypes.Type(value = JoinOperatorInfo.class, name = "joinOperatorInfo"),
         @JsonSubTypes.Type(value = WindowInfo.class, name = "windowInfo"),
-        @JsonSubTypes.Type(value = TableWriterInfo.class, name = "tableWriter")})
+        @JsonSubTypes.Type(value = TableWriterInfo.class, name = "tableWriter"),
+        @JsonSubTypes.Type(value = TaskOutputOperator.TaskOutputOperatorInfo.class, name = "taskOutputOperatorInfo")})
```

Comment:
move below `PartitionedOutputInfo`.

nit: I think we probably should use `io.trino.operator.OperatorContext#metrics` for that rather than custom info, but we can keep it for now. With `Info` engine summarize non-final stats, so coordinator-traffic is reduced during query execution